app/controllers/api/v1/merchants/revenue_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Merchants::RevenueController < ApplicationController
+  def index
+    render json: MerchantSerializer.new(Merchant.most_revenue(params[:quantity]))
+  end
+end
```

Comment:
Should I keep making controllers for each new functionality or should I make some form of aggregates controller?